mhctools/file_formats.py

```diff
@@ -39,7 +39,7 @@ def create_input_fasta_files(fasta_dictionary, max_file_records=None):
     i_range = [0] if not max_file_records else range(
         int(ceil(n_fasta_records / max_file_records)))
     input_files = [tempfile.NamedTemporaryFile(
-        "w_%s" % i, prefix="peptide", delete=False) for i in i_range]
+        "w", prefix="peptide", delete=False) for i in i_range]
```

Comment:
Also is there a reason the `mode` is set to `w` as opposed to the default `w+b`?

> mode parameter defaults to 'w+b' so that the file created can be read and written without being closed